slothy/targets/arm_v7m/arch_v7m.py

```diff
@@ -0,0 +1,1882 @@
+import logging
+import inspect
+import re
+import math
+from enum import Enum
+from functools import cache
+
+from slothy.helper import SourceLine, Loop
+from sympy import simplify
+
+llvm_mca_arch = "arm"  # TODO
+
+
+class RegisterType(Enum):
+    GPR = 1
+    FPR = 2
+    STACK_FPR = 3
```

Comment:
Those are relicts from the AArch64 model, right? We don't seem to need them.

Reply:
Should be fixed.